bagua/torch_api/distributed.py

```diff
@@ -60,25 +60,52 @@ def __init__(
             self.parameters_to_ignore = []
 
     def unwarp(self):
+        """
+        Return the unwarmped module.
```

Comment:
typo

Reply:
also the function name should be unwrap

Reply:
OK，this has been modified locally

Reply:
let me know when all conversations are resolved so that I can start another round of review

Reply:
ok